starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -1177,9 +1192,9 @@ private void maybeSetMaxInputSize(MediaFormat format) {
         return;
     }
     // Estimate the maximum input size assuming three channel 4:2:0 subsampled input frames.
-    int maxInputSize = (maxPixels * 3) / (2 * minCompressionRatio);
-    format.setInteger(MediaFormat.KEY_MAX_INPUT_SIZE, maxInputSize);
     try {
+      int maxInputSize = (maxPixels * 3) / (2 * minCompressionRatio);
+      format.setInteger(MediaFormat.KEY_MAX_INPUT_SIZE, maxInputSize);
```

Comment:
It seems that the try-catch was for MediaFormat.getInteger(KEY_MAX_INPUT_SIZE). Is it intended to change it to catch exceptions for setInteger()?

Reply:
Moved it out of try-catch, as the try-catch is for MediaFormat.getInteger(KEY_MAX_INPUT_SIZE).